apps/ui/src/components/SpaceDelegates.vue

```diff
@@ -93,12 +94,29 @@ watchEffect(() => setTitle(`Delegates - ${props.space.name}`));
 
 <template>
   <div
-    v-if="!delegation.apiUrl || delegation.apiType === 'split-delegation'"
+    v-if="!delegation.apiUrl"
     class="px-4 py-3 flex items-center text-skin-link space-x-2"
   >
     <IH-exclamation-circle class="shrink-0" />
     <span>Invalid delegation settings.</span>
   </div>
+  <UiMessage
+    v-if="delegation.apiType === 'split-delegation'"
+    :type="'info'"
+    class="m-4"
+  >
+    This space uses the split-delegation feature, which is currently not
+    supported on the new interface. You can view the delegates dashboard on the
+    <a
+      :href="`${SNAPSHOT_URLS[props.space.network]}/#/${props.space.id}/delegates`"
+      target="_blank"
+      class="inline-flex items-center font-bold"
+    >
+      previous interface
+      <IH-arrow-sm-right class="inline-block -rotate-45" />
+    </a>
+    .
```

Comment:
```suggestion
      <IH-arrow-sm-right class="inline-block -rotate-45" /></a
    >.
```

To remove extra space between icon and dot.